build.sbt

```diff
@@ -46,6 +46,8 @@ lazy val `postgresql-init-core` = (project in file("core"))
         "io.estatico" %% "newtype" % "0.4.4",
         "org.tpolecat" %% "skunk-core" % "0.2.2",
         "org.typelevel" %% "log4cats-slf4j" % "2.1.1",
+        "com.amazonaws" % "aws-lambda-java-log4j2" % "1.5.1",
+        "org.apache.logging.log4j" % "log4j-slf4j-impl" % "2.17.1",
```

Comment:
This artifact should route log events created using slf4j to the log4j2 backend, so they'll end up in CloudWatch Logs instead of being dropped.